wifi.go

```diff
@@ -0,0 +1,16 @@
+package drivers
+
+import (
+	"errors"
+	"time"
+)
+
+var (
+	ErrWiFiMissingSSID    = errors.New("missing SSID")
+	ErrWiFiConnectTimeout = errors.New("WiFi connect timeout")
+)
+
+type WiFiDriver interface {
+	ConnectToAccessPoint(ssid, pass string, timeout time.Duration) error
+	Disconnect() error
+}
```

Comment:
It seems that the following can also be included in the interface.
At the moment, espat has a slightly different interface, but I think it can be unified.

```
GetIP() (ip, subnet, gateway IPAddress, err error)
```

https://github.com/tinygo-org/drivers/blob/90a13a697bc51b41b878ffc9b0fb5ff201ef0731/wifinina/wifinina.go#L513-L521
https://github.com/tinygo-org/drivers/blob/90a13a697bc51b41b878ffc9b0fb5ff201ef0731/espat/wifi.go#L61-L65
https://github.com/tinygo-org/drivers,/blob/90a13a697bc51b41b878ffc9b0fb5ff201ef0731/rtl8720dn/wifi.go#L60-L72